test/Microsoft.SqlTools.ServiceLayer.Test.Common/RunEnvironmentInfo.cs

```diff
@@ -0,0 +1,58 @@
+//------------------------------------------------------------------------------
+// <copyright file="RunEnvironmentInfo.cs" company="Microsoft">
+//         Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+//------------------------------------------------------------------------------
+using System;
+using System.IO;
+using System.Reflection;
+using Microsoft.SqlTools.ServiceLayer.Test.Commons;
+
+namespace Microsoft.SqlTools.ServiceLayer.Test.Common
+{
+    public class RunEnvironmentInfo
+    {
+        private static string cachedTestFolderPath;
+
+        public static bool IsLabMode()
+        {
+            string bvtLabRoot = Environment.GetEnvironmentVariable(Consts.BVTLocalRoot);
+            if (String.IsNullOrEmpty(bvtLabRoot))
+            {
+                return false;
+            }
+            return true;
+        }
+
+        /// <summary>
+        /// Location of all test data (baselines, etc).
+        /// </summary>
+        /// <returns>The full path to the test data directory</returns>
+        public static string GetTestDataLocation()
+        {
+            string testFolderPath;
+            string testPath = @"test\Microsoft.SqlTools.ServiceLayer.IntegrationTests";
+            string projectPath = Environment.GetEnvironmentVariable(Consts.ProjectPath);
+
+            if (projectPath != null)
+            {
+                testFolderPath = Path.Combine(projectPath, testPath);
+            }
+            else
+            {
+                if (cachedTestFolderPath != null)
+                {
+                    testFolderPath = cachedTestFolderPath;
+                }
+                else
+                {
+                    string defaultPath = Path.Combine(typeof(Scripts).GetTypeInfo().Assembly.Location, @"..\..\..\..\..");
+                    testFolderPath = Path.Combine(defaultPath, @"Microsoft.SqlTools.ServiceLayer.IntegrationTests");
```

Comment:
I suspect the file should end up looking as follows. I'm working on a task that needs this, so it's helping me see what's needed:
```
//
// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.
//

using System;
using System.IO;
using System.Reflection;

namespace Microsoft.SqlTools.ServiceLayer.Test.Common
{
    public class RunEnvironmentInfo
    {
        private static string cachedTestFolderPath;

        public static bool IsLabMode()
        {
            string bvtLabRoot = Environment.GetEnvironmentVariable(Constants.BVTLocalRoot);
            if (string.IsNullOrEmpty(bvtLabRoot))
            {
                return false;
            }
            return true;
        }

        /// <summary>
        /// Location of all test data (baselines, etc).
        /// </summary>
        /// <returns>The full path to the test data directory</returns>
        public static string GetTestDataLocation()
        {
            string testFolderPath;
            string testPath = @"test\Microsoft.SqlTools.ServiceLayer.Common";
            string projectPath = Environment.GetEnvironmentVariable(Constants.ProjectPath);

            if (projectPath != null)
            {
                testFolderPath = Path.Combine(projectPath, testPath);
            }
            else
            {
                if (cachedTestFolderPath != null)
                {
                    testFolderPath = cachedTestFolderPath;
                }
                else
                {
                    // We are running tests locally, which means we expect to be running inside the bin\debug\netcoreapp directory
                    // Test Files should be found at the root of the project so go back the necessary number of directories for this 
                    // to be found. We are manually specifying the testFolderPath here for clarity on where to expect this
                    string defaultPath = Path.Combine(typeof(Scripts).GetTypeInfo().Assembly.Location, @"..\..\..\..\..");
                    testFolderPath = Path.Combine(defaultPath, @"Microsoft.SqlTools.ServiceLayer.Common");
                    cachedTestFolderPath = testFolderPath;
                }
            }
            return testFolderPath;
        }

    }
}
```